src/split.c

```diff
@@ -59,6 +65,7 @@ rcutils_split(
       ++string_array->size;
     }
   }
+  // TODO(wjwwood): refactor this function so it can use rcutils_string_array_init() instead
```

Comment:
No, I wasn't planning on it. The split functions logic involves resizing the `size` argument dynamically as it goes. I didn't want to figure out how to change it so it could figure out the end size once and not need to change storage as it goes.